rdrf/rdrf/templates/registration/login_assistance_can_not_verify.html

```diff
@@ -0,0 +1,14 @@
+{% extends "rdrf_cdes/base.html" %}
+{% load i18n %}
+{% load static from staticfiles %}
+
+{% block content %}
+<div class="row">
+    <div class="col-md-6 col-md-offset-3">
+        <div class="alert alert-warning">
+            {% trans "Unfortunately we don't have the required information to verify your identity. Please contact the registry owners for assistance." %}
```

Comment:
Presumably a patient, say, would have email of the owners or would have enough info in initial registration emails to locate the splashcreen (?) with that info. But we should check.